### ISSUE TITLE
Call to undefined method Sportmonks\SoccerAPI\Requests\LiveScore::setLeagues()

### ISSUE THREAD

**Issue by deathwebo** [sebastiaanspeck/sportmonks#25]
**Describe the bug**
An error occur when you try to access the livescore and fixture by date urls. This appear to be happening because the method `setLeagues` doesn't exist anymore in the SoccerAPIClient objects

**To Reproduce**

1. Spin a local instance with `php artisan serve`
2. Click on the `livescore` or `today` link
3. See error

**Expected behavior**
The requested pages should be loaded correctly

**Screenshots**
![screen shot 2018-10-15 at 8 45 29 am](https://user-images.githubusercontent.com/8619982/46961939-ba876380-d056-11e8-87e5-c186ada1ce3a.png)

**Desktop (please complete the following information):**
 - OS: macOS 10.13.6
 - Browser Google Chrome
 - Version 69.0.3497.100

**Comment by sebastiaanspeck** [comment]
I changed the LiveServiceProvider recently. Try changing the ‘use ...\view’ to ‘use View’ and it should work

**Comment by deathwebo** [comment]
Ok! Or I can grab your latest changes and try again.

**Comment by sebastiaanspeck** [comment]
Hmm. Do other pages work?

**Comment by deathwebo** [comment]
Yes, only the controller actions in which that method was used weren't working.

**Comment by sebastiaanspeck** [comment]
Strange. By me it's working properly..

**Comment by sebastiaanspeck** [comment]
http://127.0.0.1:8000/leagues/208 is working with the most recent code? (so not your local code)
